tools/azure-sdk-tools/devtools_testutils/__init__.py

```diff
@@ -2,10 +2,12 @@
 from .azure_testcase import AzureTestCase, is_live
 from .resource_testcase import (FakeResource, ResourceGroupPreparer)
 from .storage_testcase import (FakeStorageAccount, StorageAccountPreparer)
+from .keyvault_testcase import KeyVaultAccountPreparer
 
 __all__ = [
     'AzureMgmtTestCase', 'AzureMgmtPreparer',
     'FakeResource', 'ResourceGroupPreparer',
     'FakeStorageAccount', 'StorageAccountPreparer',
-    'AzureTestCase', 'is_live'
+    'AzureTestCase', 'is_live',
+    "KeyVaultAccountPreparer"
```

Comment:
If you do that you need to update the setup.py to add azure-mgmt-keyvault, otherwise it will crash with people that doesn't use it